tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "forceConsistentCasingInFileNames": true,
     "baseUrl": ".",
     "paths": {
-      "~/*": ["./app/*"]
+      "~/*": ["./app/*"],
+      "content-collections": ["./.content-collections/generated"]
```

Comment:
Please use a more explicit alias over here. Something like `#content-collections`? or something similar?

Reading through the PR, I honestly mistook the `import { allPosts } from 'content-collections'` as a normal package.

Reply:
This may not be possible. Looking at the [source](https://github.com/sdorra/content-collections/blob/31664af5091fe0024bdb733c55a6810a80d43fc7/packages/vite/src/index.ts#L57) for content-collections, it looks like their Vite plugin explicitly sets it up to use `'content-collections'` are the alias.